config/default.yml

```
@@ -3,6 +3,7 @@ Minitest:
   Include:
     - '**/test/**/*'
     - '**/*_test.rb'
+  DocumentationBaseURL: https://docs.rubocop.org/rubocop-minitest
```

That's just my two cents.

```suggestion
  DocumentationBaseURL: https://docs.rubocop.org/rubocop-minitest
  Include:
    - '**/test/**/*'
    - '**/*_test.rb'
```

---

@r7kamura Ping :-)